### PR TITLE
Wait for config propagation when adding shared service to catalog

### PR DESCRIPTION
**Pull request by chemikadze**

### CODE REVIEW

File: qubell/api/private/service.py [hunk 1]
```diff
@@ -47,6 +48,11 @@
 
 SHARED_INSTANCES_PARAMETER_NAME = system_application_parameters[SHARED_INSTANCE_CATALOG_TYPE]
 
+def _field_or_identity(obj, field):
```
Comment (dmakhno) [inline qubell/api/private/service.py]: where is the usage?

**Comment by vasichkin** [comment]
Let's try it

File: qubell/api/private/service.py [hunk 2]
```diff
@@ -66,6 +72,15 @@ def add_shared_instance(self, revision, instance):
         payload = json.dumps({"revisionNameId": revision.nameId,
                               "instanceId": instance.instanceId})
         self._router.post_instance_shared(org_id=self.organizationId, env_id=instance.environment.id, data=payload)
+        @retry(5, 1, 2)
+        def projection_update_monitor():
+            return self.get_shared_instance_id(revision) == instance.instanceId
```
Comment (dmakhno) [inline qubell/api/private/service.py]: did you plan to call `projection_update_monitor`?
this code seems does nothing.

Comment (chemikadze) [inline qubell/api/private/service.py]: shi~

**Comment by dornatsky** [comment]
okay

**Pull request closed without merging**